packages/components/src/components/action-bar/action-bar.tsx

```diff
@@ -361,6 +399,12 @@ export class ActionBar extends LitElement {
     this.actions = Array.from(this.el.querySelectorAll("calcite-action"));
   }
 
+  private updateTabIndexOfItems(target: Action["el"]): void {
```

Comment:
Can you revisit the name of this method and argument to better convey the intent?